search-commons/src/test/java/no/unit/nva/search/IndexDocumentWrapperLinkedDataTest.java

```diff
@@ -37,8 +37,22 @@
     @Test
     public void toFramedJsonLdReturnsJsonWithValidReferenceData() throws Exception {
 
-        final URI uri = getRandomUri();
+        // https://github.com/jsonld-java/jsonld-java
```

Comment:
A bit commenty here

Reply:
Blueprint for the other solution.... Removed